exercises/sublist/sublist_test.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'sublist'
+
+class Sublist < Minitest::Test
```

Comment:
I would name the test class `SublistTest` rather than Sublist, for a couple of reasons:

1. It follows the convention that we use elsewhere.
2. It won't risk conflicting with the implementation.